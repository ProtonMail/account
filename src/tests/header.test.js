import render from 'preact-render-to-string';

import Header from '../components/header';

describe('Initial Test of the Header', () => {
    test('Header renders 3 nav items', () => {
        const tree = render(<Header />);
        expect(tree).toMatchSnapshot();
    });
});
