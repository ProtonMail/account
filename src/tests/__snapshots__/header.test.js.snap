// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Initial Test of the Header Header renders 3 nav items 1`] = `
<header class="header">
  <h1>Proton Account</h1>
  <nav>
    <a href="/" class="active">Login</a>
  </nav>
</header>
`;
